src/main/java/com/cinema/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.cinema.dao.impl;
+
+import com.cinema.dao.OrderDao;
+import com.cinema.exceptions.DataProcessingException;
+import com.cinema.lib.Dao;
+import com.cinema.model.Order;
+import com.cinema.model.User;
+import com.cinema.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order add(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.persist(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert Order entity " + order.toString(), e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't insert Order entity " + order, e);
```
this is redundant - toString() is called implicitly

Reply:
done